src/map/status.hpp

```diff
@@ -3172,7 +3172,7 @@ struct status_data {
 #endif
 		matk_min, matk_max,
 		speed,
-		amotion, adelay, dmotion;
+		amotion, clientamotion, adelay, dmotion;
```

Comment:
```suggestion
		amotion, client_amotion, adelay, dmotion;
```
If `clientamotion` is too hard to read without snake-case then feel free to adjust it!

Reply:
I think it's fine.